src/Service/PubliccodeService.php

```diff
@@ -0,0 +1,836 @@
+<?php
+
+namespace OpenCatalogi\OpenCatalogiBundle\Service;
+
+use App\Entity\Entity;
+use App\Entity\Gateway as Source;
+use App\Entity\Mapping;
+use App\Entity\ObjectEntity;
+use CommonGateway\CoreBundle\Service\CacheService;
+use CommonGateway\CoreBundle\Service\CallService;
+use CommonGateway\CoreBundle\Service\GatewayResourceService;
+use CommonGateway\CoreBundle\Service\HydrationService;
+use CommonGateway\CoreBundle\Service\MappingService;
+use App\Service\SynchronizationService;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use GuzzleHttp\Exception\ClientException;
+use phpDocumentor\Reflection\Types\This;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Serializer\Encoder\XmlEncoder;
+use Symfony\Component\Serializer\Encoder\YamlEncoder;
+
+class PubliccodeService
```

Comment:
Missing class code block (maybe double check if you missed this in any other services)